corehq/blobs/__init__.py

```diff
@@ -83,6 +83,7 @@ class CODES:
     fixture = 13        # domain-fixtures
     demo_user_restore = 14  # DemoUserRestore
     data_file = 15      # domain data file (see DataFile class)
+    form_multimedia = 16     # form submission multimedia zip
```

Comment:
@millerdev is this appropriate or should I be using `CODES.temp`?

Reply:
I think either is fine. If these are frequently huge then it may be useful to have a dedicated code for them.